src/Winium.Desktop.Driver/CommandExecutors/NewSessionExecutor.cs

```diff
@@ -32,6 +31,14 @@ protected override string DoImpl()
             // Gives sometime to load visuals (needed only in case of slow emulation)
             Thread.Sleep(this.Automator.ActualCapabilities.LaunchDelay);
 
+            // Update running application process pointing to the real process instead of the launcher in such cases
+            if ((this.Automator.Application.HasExited()) && (this.ExecutedCommand.Parameters["desiredCapabilities"]["realProcessName"] != null))
```

Comment:
@zelonght Fixing as this suggestion will make source code is as same as my code before :)